test/test_modules.py

```diff
@@ -359,8 +359,12 @@ def _can_be_noncontiguous(obj):
             # === Forward with default input
             with freeze_rng_state():
                 default_output = m(*input_args, **input_kwargs)
-                grad_output = default_output.clone().detach_().normal_()
-                default_output.backward(grad_output, retain_graph=True)
+                if isinstance(default_output, torch.Tensor):
+                    grad_output = default_output.clone().detach_().normal_()
+                    default_output.backward(grad_output, retain_graph=True)
+                else:
+                    grad_output = tuple(o.clone().detach_().normal_() for o in default_output)
+                    (o.backward(grad_output, retain_graph=True) for o in grad_output)
```

Comment:
Looks like this logic was added because `test_non_contiguous_tensors` did not support multiple outputs - is that correct?

Is it possible to use `_traverse_obj()` defined above to simplify this?

Reply:
Had to tweak this slightly as it was always returning `None`. Now it works correctly. I don't think `_traverse_obj` will work with this.